docs/schema/accesscontrolentry-element-in-accesscontrollist-bdcmetadata-schema.md

```diff
@@ -56,47 +56,16 @@ The following sections describe attributes, child elements, and parent elements.
 </table>
 
 ### Child elements
-
-<table>
-<colgroup>
-<col width="30%" />
-<col width="70%" />
-</colgroup>
-<thead>
-<tr class="header">
-<th align="left"><p>Element</p></th>
-<th align="left"><p>Description</p></th>
-</tr>
-</thead>
-<tbody>
-<tr class="odd">
-<td align="left"><p><span sdata="link"><a href="right-element-in-accesscontrolentry-bdcmetadata-schema.md">Right element in AccessControlEntry (BDCMetadata Schema)</a></span></p></td>
-<td align="left"><p>A **Right** element that specifies the permissions available to the security principal.</p></td>
-</tr>
-</tbody>
-</table>
+  
+| Element | Description |
+| --- | --- |
+| [Right element in AccessControlEntry (BDCMetadata Schema)](right-element-in-accesscontrolentry-bdcmetadata-schema.md) | A **Right** element that specifies the permissions available to the security principal. |
 
 ### Parent elements
-
-<table>
-<colgroup>
-<col width="30%" />
-<col width="70%" />
-</colgroup>
-<thead>
-<tr class="header">
-<th align="left"><p>element</p></th>
-<th align="left"><p>Description</p></th>
-</tr>
-</thead>
-<tbody>
-<tr class="odd">
-<td align="left"><p><span sdata="link"><a href="accesscontrollist-element-bdcmetadata-schema.md">AccessControlList element (BDCMetadata Schema)</a></span></p></td>
-<td align="left"><p>The access control list (ACL) that contains this ACE.</p></td>
-</tr>
-</tbody>
-</table>
-
+  
+| element | Description |
```

Comment:
Even though the source doesn't have this capitalized, let's correct that oversight.